src/main/webapp/resources/js/apis/cart/cart.js

```diff
@@ -1,10 +1,24 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { notification } from "antd";
 import axios from "axios";
-import { setBaseUrl } from "../../utilities/url-utilities";
 import { cartUpdated } from "../../utilities/events-utilities";
-import { notification } from "antd";
+import { setBaseUrl } from "../../utilities/url-utilities";
 
 const AJAX_URL = setBaseUrl(`/ajax/cart`);
 
```

Comment:
Add comments

Reply:
Updated in 011b3a28f9aa96d9fc62d39e61123f175e18faea